internal/must/must.go

```diff
@@ -114,6 +115,14 @@ func NotEqualStr(t *testing.T, got, want, msg string) {
 	}
 }
 
+// StartWithStr ensures that got starts with wantPrefix else logs an error.
+func StartWithStr(t *testing.T, got, wantPrefix, msg string) {
+	t.Helper()
+	if !strings.HasPrefix(got, wantPrefix) {
+		t.Errorf("StartWithStr: #{msg}: got '#{got}' without prefix '#{wantPrefix}'")
```

Comment:
I'm unfamiliar with this param structure, does this actually work?

Reply:
it's used in the surrounding code, so I hope so :p

Reply:
Uhh, where exactly?

Reply:
excellent question. wtf am I smoking